### ISSUE TITLE
Error when selecting "Icons Enable"

### ISSUE THREAD

**Issue by camijohn515** [robertohuertasm/vscode-icons#26]
I installed the extension with no issues.  I launched Visual Studio Code as Administrator and then I hit F1 on the keyboard.  Then I typed  Icons and the following selections came up:
Icons Enable
Icons Disable
Icons Update

As soon as I select "Icons Enable"  I get the following message:
Error No handler found for the command: 'extension.installIcons'. An extension might be missing an activation event.

Any help will be greatly appreciated.

Thanks!

**Comment by robertohuertasm** [comment]
I guess you're using Windows... What VSCode version are you using? Have you tried to reinstall the extension? It's weird, this error is because somehow the activation process never happens and it should be executed on opening VSCode. Try reinstalling the extension.

**Comment by camijohn515** [comment]
Yes, I am using windows 7 Pro.  I tried re-installing the extension several times on 2 different machines.  With the same results.  Other extensions are installing fine.  I am using Visual Studio 1.2.1.  I installed that version today.  I tried to reinstall the extension before and after the 1.2.1 update.

**Comment by robertohuertasm** [comment]
Those 2 different machines were also Windows 7?

**Comment by camijohn515** [comment]
Actually, one of the machines is a Windows 2008 server. VS Code and other extensions run fine on it.  I'm not sure if it's supposed to be compatible with that OS or not.  I am going to install VS Studio Code on a laptop that is not on the network and see what happens.

**Comment by robertohuertasm** [comment]
are you behind some firewall? The extension will try to download the icons from the Github repository. Try enabling the Developer Tools in your VSCode and see if something is thrown into the console while enabling the icons.

**Comment by camijohn515** [comment]
I enabled the developer options and I figured out. It could not find the package.json file inside of the extension's folder.  For whatever reason, it failed to download.  I manually downloaded the zip file, extracted it and copied the folder to the extensions directory and it works great. Like you said, it looks like some kind of firewall was blocking it.  You can close this issue. Thanks for your help!  Great extension.

**Comment by Adham-Hajji** [comment]
Hello.

Sorry for the issue debunking. I installed vscode-icons some time ago, and I tried to configure some custom bindings via the workspace settings. The JSON chunk of code was the following:

```json
"settings": {
  "vsicons.associations.folders": [
    { "icon": "www", "extensions": "sites", "format": "svg" }
  ]
}
```

In order to make those change appear on the editor, I followed the documentation: I pressed the `F1` button, then I tried to run the Activate command of the extension.

Unfortunately, I encountered the same issue as @camijohn515 . VSCode triggered this error:

> command 'vscode-icons.activateIcons' not found

I have my Visual Studio Code running as an administrator, and as @camijohn515 did I enabled Devtools and checked the Chrome console. Oddly, there isn't any error concerning a file missing. Instead, the console shows this exception:

```console
command 'vscode-icons.activateIcons' not found
e.doShow @ /C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:2925
````

Thank you for your help. :)
